courses/models_test.py

```diff
@@ -18,6 +20,12 @@ def test_tostring(self):
         """
         assert str(Course(title='test')) == 'test'
 
+    def test_towebhook(self):
+        """
+        test to_webhook implementation returns valid json object
+        """
+        json.dumps(CourseFactory.build().to_webhook())
```

Comment:
Can you assert that `module.to_webhook()['external_pk']` is a `str` like you do in the `Module` test below?